webapp/README

```diff
@@ -30,6 +30,15 @@ entry.js: the entry point into the app
   for all variables. So now its half-and-half, sorry :(. It shouldn't be that
   hard to fix...
 
+- You can use the import script to automatically write your imports for you!
+  Its much easier than writing your includes yourself, even if you have to
+  modify/fix up the script for edge cases that it doesn't handle.
```

Comment:
What edge cases are not handled? Having to figure out edge cases sounds a little annoying...

Does the script take any command line arguments or whatever? If so, can you put an example command line here?